libs/wingsdk/src/core/utility-types.ts

```diff
@@ -14,13 +14,23 @@ export type PickNonFunctions<T> = Pick<
 /**
  * Represents a map of liftable objects with string keys.
  */
-export type LiftableMap = { [key: string]: Liftable };
+export type LiftableHash = { [key: string]: Liftable };
```

Comment:
Why were these renames done?

nit: imo `LiftableHash` is a slightly worse name. I would say `LiftableHashMap` or `LiftableRecord`

Reply:
OK, changed to `LiftableRecord`